### DISCUSSION TITLE
Proxmox 8.2.4 Update Issues

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#3225] [Announcements]
Update via the GUI using the test repository, there are issues with the no subscription repo.
<https://forum.proxmox.com/threads/upgrading-pve-tries-to-remove-proxmox-ve-package.149101/>

**Comment by Mellowlynx** [comment]
Thanks for the heads up.

**Comment by tteck** [comment]
The no-subscription repository gets its updates released from the test repository; someone forgot to release all of the necessary files to complete the upgrade. After getting the "you are attempting to remove the metapackage proxmox-ve" message, enable the test repository and hit the refresh button and upgrade again.

**Comment by tteck** [comment]
The no-subscription repository has been fixed.
